checkov/common/util/type_forcers.py

```diff
@@ -115,3 +115,22 @@ def convert_csv_string_arg_to_list(csv_string_arg: list[str] | str | None) -> li
         return csv_string_arg[0].split(',')
     else:
         return csv_string_arg
+
+
+def convert_prisma_policy_filter_to_dict(filter_string: str) -> Dict[Any, Any]:
+    """
+    Converts the filter string to a dict. For example:
+    'policy.label=label,cloud.type=aws' becomes -->
+    {'policy.label': 'label1', 'cloud.type': 'aws'}
+    Note that the API does not accept lists https://prisma.pan.dev/api/cloud/cspm/policy#operation/get-policies-v2
+    This is not allowed: policy.label=label1,label2
+    """
+    filter_params = {}
+    if filter_string:
+        try:
+            for f in filter_string.split(','):
+                f_name, f_value = f.split('=')
+                filter_params[f_name] = f_value
```

Comment:
@kartikp10 I'd expect to see some `trim()` here, to avoid stupid user input mistakes... Example bad input:
`'policy.label=label, cloud.type=aws'`

Can be done in another PR